src/_posts/platform/deployment/buildpacks/2000-01-01-apt.md

```diff
@@ -66,8 +66,9 @@ http://downloads.sourceforge.net/project/wkhtmltopdf/0.12.1/wkhtmltox-0.12.1_lin
 ```
 
 {% warning %}
-Current base image is based on **Ubuntu 18:04 Bionic**, you have to make sure
-the chosen packages are compatible with this release. See [this blog article](https://scalingo.com/articles/2019/08/06/multiple-stacks.html) to see how to change your stack.
+You have to make sure the chosen packages are compatible with the release the stack image.
```

Comment:
I think there is a confusion in your wording between "stack", "release", "version", "image", etc. I would re-use the wording from the Stacks page (https://doc.scalingo.com/platform/internals/stacks/stacks). 

```suggestion
You have to make sure the chosen packages are compatible with the stack base distribution.
```